src/TwigComponent/src/DependencyInjection/TwigComponentExtension.php

```diff
@@ -219,6 +220,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->info('Enables the profiler for Twig Component (in debug mode)')
                     ->defaultValue('%kernel.debug%')
                 ->end()
+                ->booleanNode('profiler_dump_components')
+                    ->info('Enables the dump components for Twig Component (in debug mode)')
+                    ->defaultValue('%kernel.debug%')
+                ->end()
```

Comment:
defaultTrue() seems more logical here .. (its not just in debug mode but... when twig component profiler is enabled) 

Without the parenthesis, probably something like 

"Dump components in the Twig Component profiler panel" 

wdyt ?